test/unit/utils/page.spec.js

```diff
@@ -112,8 +112,34 @@ describe('Page Utils', () => {
 
     expect(getReferrer(iframe2.contentWindow)).to.be.eql('https://first.example.com?key=value')
   })
+
+  it('getContextElements should return empty when contextSelectors is empty', function () {
+    createElement('p', 'mailTo: john@test.com, also found: another@test.com !', document)
+    expect(getContextElements('', 1000)).to.be.eql('')
+  })
+
+  it('getContextElements should truncate the maximum lentgh of the encoded context elements', function () {
+    createElement('h1', 'mailTo: john@test.com, also found: another@test.com !', document)
+    const result = getContextElements('h1', 10).length
+    expect(result).to.be.eql(10)
+  })
+
+  it('getContextElements should properly encode the context elements found', function () {
+    createElement('p', 'Some dummy text', document)
+    const size = getContextElements('', 1000).length
+    // Let's simply check if the format is ok
+    expect(size % 4).to.be.eql(0)
```

Comment:
We do not ensure this anywhere, we just do `base64UrlEncode(collectedElements).slice(0, contextElementsLength)`.

Reply:
Updating as per the previous comment